configs/body3d/simple_baseline/h36m/linear_h36m_block2x1024_maxnorm.py

```diff
@@ -0,0 +1,136 @@
+log_level = 'INFO'
+load_from = None
+resume_from = None
+dist_params = dict(backend='nccl')
+workflow = [('train', 1)]
+checkpoint_config = dict(interval=10)
+evaluation = dict(interval=10, metric='mpjpe', key_indicator='MPJPE')
```

Comment:
How to know when to use the lower case and when to use the upper case, of mpjpe

Reply:
Key indicators will be changed to lower case in eval_hook.

Reply:
good